src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -852,6 +821,24 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
 
                 jobScheduler.pushJob(jobDetails);
                 JobResponse jobResponse = jobScheduler.getResult(jobDetails);
+                std::string errorMessage = jobResponse.getParameter(Conts::PARAM_KEYS::ERROR_MESSAGE);
+
+                if (!errorMessage.empty()) {
+                    string error_message = "Cannot process request now. Please try again later.";
```

Comment:
When we get an error message here, we should send a meaningful response back to the user via the frontend service session. Just logging an error message here does not help.

Reply:
Fixed this by returning the appropriate error message

Reply:
Did you test what kind of error messages you get here? Eventhough you route the system generated error message to user, it may not be meaningful. Sometimes the previous error message we returned to user `Cannot process request now. Please try again later.` could have been better error message here. Therefore, please test what error messages you get and list those error messages as a reply to this comment.